Sprint-3/revise/implement/get-ordinal-number.js

```diff
@@ -0,0 +1,34 @@
+function getOrdinalNumber(num) {
+    //function will add the correct suffix to the appropriate number with certain conditions
+    let ordinal = "";
+    if (num > 0) {
+        let StringNum = num.toString();
```

Comment:
Common practice is to begin a variable name with lowercase letter.

Reply:
A mistake on my part, I've fixed it